Source/SimpleActions/Public/Structs/SimpleActionList.h

```diff
@@ -20,8 +20,13 @@ struct SIMPLEACTIONS_API FSimpleActionList
 	
 public:
 
-	UPROPERTY(EditAnywhere, BlueprintReadOnly, Instanced, meta = (ShowOnlyInnerProperties))
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Instanced)
 	TArray<TObjectPtr<USimpleAction>> Actions;
 
+	FSimpleActionList() : FSimpleActionList(TArray<USimpleAction*>()) {}
+	FSimpleActionList(TArray<USimpleAction*> Actions) {
```

Comment:
I think it should be
```cpp
FSimpleActionList(TArray<USimpleAction*> NewActions) : Actions(NewActions) {}
```

Reply:
Doesn't have to be New Actions though, could be an edited array of actions that were already present.
So I would disagree.

Reply:
the constructor is only called for making a new object. the `new` was to avoid name conflicting, might not be necessary.

Reply:
Isn't it also called for a UE auto generated make node?